### DISCUSSION TITLE
How to setup and work with RSS

### DISCUSSION THREAD

**Discussion by vyruz** [haugene/docker-transmission-openvpn#1979] [Q&A] [answered]
Does anyone know how to get RSS setup properly, and after that how you are adding or changing elements for the feed? I had tried adding the RSS_URL and RSS_REGEX like below but this gave me errors. Even removing the REGEXP field completely seemed to cause errors which resulted in the RSS container to exit. 

 rss:
  image: haugene/transmission-rss
  links:
    - transmission
  environment:
    - RSS_URL=https://nyaa.si/?page=rss
    - RSS_REGEXP=[SubsPlease] - Sonny Boy.* (720).*

**Comment by pkishino** [comment, marked as answer]
http://haugene.github.io/docker-transmission-openvpn/rss-plugin/

**Reply by vyruz** [reply]
I didn't find that page to be all that helpful. I dont start my containers with run i use docker compose and have it run through docker desktop on a windows machine so i'm not sure where a transmission-rss.conf would be placed as there isnt one by default. That still also leaves the question of how you add or remove from the feed. Do you have to restart/recompose the container if you want to add or remove a regexp?

**Reply by pkishino** [reply]
the transmission rss is hardly used by anyone these days, just so much easier to use radarr/sonarr etc.. so no further updates for it are planned.
the rss.conf you create somewhere on your disk and mount it as a volume to the docker container. easily done in compose
just add a volume:
```yaml
 volumes:
    - <transmission-rss.conf>:/etc/transmission-rss.conf
```
if you don't create and mount the file, it gets autocreated in /etc..then you can just edit it inside the container..but if you recreate or remove the container of course this get's lost, so mounting your own is recommended.
this is just a wrapper around
https://github.com/nning/transmission-rss
so for usage etc check there please.
according to this:
https://github.com/nning/transmission-rss/issues/7
it might work to notice changes without having to reload anything..

**Reply by vyruz** [reply]
Thank you for this. I havent heard of radarr/sonarr before and didn't know they worked with this kind of transmission container so ill have to look into those
